Themes/default/css/jquery.sceditor.default.css

```diff
@@ -49,6 +49,9 @@ code::before {
 	top: -1.35em;
 	left: 0;
 }
+code[data-title]::before {
+	content: 'Code: (' attr(data-title) ')';
```

Comment:
Might be an issue with localization, perhaps it will be better to move  the "Code:" part to a js var filled with its corresponding translation.

Reply:
I will address that localization  problem in the update to SCE v3